### PR TITLE
Tweaks/clarifications to search spec

### PR DESCRIPTION
**Pull request by richvdh**

### CODE REVIEW

File: specification/modules/search.rst [hunk 1]
```diff
@@ -8,8 +8,6 @@ rooms that the user has been in, including those that they have left. Only
 events that the user is allowed to see will be searched, e.g. it won't include
 events in rooms that happened after you left.
 
-Client behaviour
-----------------
```
Comment (Kegsay) [inline specification/modules/search.rst]: For consistency with other modules, please keep this in. See https://github.com/matrix-org/matrix-doc/blob/rav/search_spec_fixes/specification/modules/_template.rst

File: api/client-server/search.yaml [hunk 2]
```diff
@@ -20,12 +20,19 @@ securityDefinitions:
...
         Performs a full text search across different categories.
       security:
         - accessToken: []
       parameters:
+        - in: query
+          name: next_batch
+          type: string
+          description: |-
+            The point to return events from. If given, this should be a
+            `next_batch` result from a previous call to this endpoint.
```
Comment (Kegsay) [inline api/client-server/search.yaml]: There's no way to do a `prev_batch` to navigate back where you came?

**Comment by Kegsay** [comment]
LGTM other than comments.

File: specification/modules/search.rst (hunk 1 above)
Comment (richvdh) [inline specification/modules/search.rst]: OK. I've reinstated it, along with some words which make it seem slightly less out of place.

File: api/client-server/search.yaml (hunk 2 above)
Comment (richvdh) [inline api/client-server/search.yaml]: there is not.

**Comment by richvdh** [comment]
PTAL?

**Comment by Kegsay** [comment]
LGTM

**Comment by richvdh** [comment]
ta

**Pull request merged**